FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/Models/QueueType.cs

```diff
@@ -0,0 +1,12 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Jobs.Models
+{
+    public enum QueueType : byte
+    {
+        FhirToDataLake = 1,
```

Comment:
Trying to understand why the enum value is start from `1`. Is this behaviour meant to avoid unconsciously using the default value `0` to represent meaningful action?